bin/check-redis-memory-percentage.rb

```diff
@@ -76,6 +76,12 @@ class RedisChecks < Sensu::Plugin::Check::CLI
          description: 'Critical instead of warning on connection failure',
          default: false
 
+  option :conn_failure_status,
+         long: '--conn-failure-status EXIT_STATUS',
+         description: 'Exit status for Redis connection failures',
```

Comment:
This could be improved a bit to say "Returns the following exit status for Redis connection failures" which seems a bit more descriptive and does not me to jump to the code to understand what you mean.